tests/test_odl.py

```diff
@@ -2090,3 +2093,135 @@ def test_odl_reaper_removes_expired_licenses(self):
         # 6.2. Ensure that number of licenses is still 0.
         assert imported_pool.licenses_owned == 0
         assert imported_pool.licenses_available == 0
+
+    @freeze_time("2021-01-01T00:00:00+00:00")
+    def test_odl_reaper_removes_expired_licenses_with_multiple_available(self):
+        """Ensure ODLExpiredItemsReaper removes expired licenses."""
+        # 1.1. Import the test feed with an ODL license that is still valid. Set the number of licenses
+        # available to 5 to make sure that the licenses are not available once the license expires.
+        license_expiration_date = datetime_helpers.utc_now() + datetime.timedelta(days=1)
+        self.LICENSES_AVAILABLE = 5
```

Comment:
`available` in the ODL spec means `number of concurrent checkouts currently available`. 
I think we also should set `LICENSES_LEFT` that is `total number of checkouts left for a License` to the same number (5). It shouldn't be less than `available`. It doesn't really affect this test but it would represent a real life scenario.